var/www/modules/hunter/templates/showTracker.html

```diff
@@ -1,359 +1,400 @@
+                                            <th>Tags <span class="btn-link btn-interaction mouse_pointer"
+                                                           title="Edit Tags List" onclick="edit_tags();"><i
+                                                    class="fas fa-pencil-alt" style="color:Red;"></i></span></th>
+                                            <th>Email <span class="btn-link btn-interaction mouse_pointer"
+                                                            title="Edit Email List" onclick="edit_mails();"><i
+                                                    class="fas fa-pencil-alt" style="color:Red;"></i></span></th>
+                                        </tr>
+                                        </thead>
+                                        <tbody>
+                                        <tr>
+                                            <td>{{ tracker_metadata['type'] }}</td>
+                                            <td>{{ tracker_metadata['tracker'] }}</td>
+                                            <td>{{ tracker_metadata['date'][0:4] }}/{{ tracker_metadata['date'][4:6] }}/{{ tracker_metadata['date'][6:8] }}</td>
+                                            <td>
+                                                {% if tracker_metadata['level'] == 0 %}
+                                                    Private
+                                                {% else %}
+                                                    Global
+                                                {% endif %}
+                                            </td>
+                                            <td>{{ tracker_metadata['user_id'] }}</td>
+                                            <td>
+                                                {% if tracker_metadata['first_seen'] %}
+                                                    {{ tracker_metadata['first_seen'][0:4] }}/
+                                                    {{ tracker_metadata['first_seen'][4:6] }}/
+                                                    {{ tracker_metadata['first_seen'][6:8] }}
+                                                {% endif %}
+                                            </td>
+                                            <td>
+                                                {% if tracker_metadata['last_seen'] %}
+                                                    {{ tracker_metadata['last_seen'][0:4] }}/
+                                                    {{ tracker_metadata['last_seen'][4:6] }}/
+                                                    {{ tracker_metadata['last_seen'][6:8] }}
+                                                {% endif %}
+                                            </td>
+                                            {% if tracker_metadata['webhook'] %}
+                                                <td>
+                                                    Turned ON
```

Comment:
I think we should show the full list (like mails)

Reply:
I did this at first, but the URL is soo long, made the table UI broken, so the only useful way was saying if ON or not showing it at all